packages/prisma/migrations/20240209223121_adds_user_password/migration.sql

```diff
@@ -0,0 +1,31 @@
+/*
+Warnings:
+
+- You are about to drop the column `password` on the `users` table. All the data in the column will be lost.
+
+ */
+-- CreateTable
+CREATE TABLE
+  "UserPassword" (
+    "hash" TEXT NOT NULL,
+    "userId" INTEGER NOT NULL
+  );
+
+-- CreateIndex
+CREATE UNIQUE INDEX "UserPassword_userId_key" ON "UserPassword" ("userId");
+
+-- AddForeignKey
+ALTER TABLE "UserPassword" ADD CONSTRAINT "UserPassword_userId_fkey" FOREIGN KEY ("userId") REFERENCES "users" ("id") ON DELETE CASCADE ON UPDATE CASCADE;
+
+-- Inserts current user password into new table
+INSERT INTO
+  "UserPassword" ("hash", "userId")
+SELECT
+  u."password",
+  u.id
+FROM
+  users u;
+
+-- AlterTable
+ALTER TABLE "users"
+DROP COLUMN "password";
```

Comment:
Should we drop this column in a future PR to ensure nothing goes wrong during this migration? Just a thought i had reading this

Reply:
Yeah agreed we should do this in a two-step process.

Reply:
Valid concerns although there shouldn't be much problem. Already tested with production-like data. 

I'm going to rename the column to passwordDeprecated to avoid accidental use instead and drop it in a future release. 

That way we can have a recoverable state just in case.

Reply:
Perfect - i just know how much drama i caused last time with the org migration PR :') i want to make sure that doesnt happen again

Reply:
@sean-brydon @keithwillcode Addressed feedback. Please re-review 🙏🏽